app/views/distributions/_distribution.html.erb

```diff
@@ -49,7 +49,7 @@
           </div>
         <% end %>
         <% tab["arches"].try(:each) do |arch| %>
-        <h3 class="my-3 text-center"><%= arch["name"] %></h3>
+        <h3 class="my-3 text-center"><%= arch["name"] %> <% if arch["desc"] %>(<%= arch["desc"] %>)<% end %></h3>
```

Comment:
My Ruby is getting a bit rusty, but I'd say this is totally fine. I would only move the space inside the conditionally printed part, to eliminate a trailing space when there is no description
```suggestion
        <h3 class="my-3 text-center"><%= arch["name"] %><% if arch["desc"] %> (<%= arch["desc"] %>)<% end %></h3>
```

Reply:
Will do thank you!